src/ledger/assets/tokens.ts

```diff
@@ -65,36 +65,28 @@ export class Tokens extends Asset {
 			return "uncomparable";
 		}
 		let res = ["lt", "eq", "gt"] as const;
-		let a = this.value;
-		let b = (asset as Tokens).value;
+		let a = this.value.sort();
```

Comment:
IF this is really required we have another problem with the methods `addID` and `sub`, because these are the only functions with which the `Tokens` should be populated. Also the constructor is missing a check that the given `bigint[]` array is already sorted (or sorted when constructing).

Reply:
In short, `this.value.sort()` should never be required.